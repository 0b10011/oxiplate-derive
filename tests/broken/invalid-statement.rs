use oxiplate_derive::Oxiplate;

#[derive(Oxiplate)]
#[oxiplate_inline = "{% thisdoesntexist %}"]
struct Data {}

fn main() {
    print!("{}", Data {});
}
