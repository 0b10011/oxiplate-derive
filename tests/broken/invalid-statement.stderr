error: Expected one of: block, endblock, if, elseif, else, endif, for, endfor
 --> tests/broken/invalid-statement.rs:4:25
  |
4 | #[oxiplate_inline = "{% thisdoesntexist %}"]
  |                         ^^^^^^^^^^^^^^^^^^
